Create page for training bazaar (#22)
eu-ssi/eussi.org: content/bazaar/_index.md

The training bazar is an activity organised by the EuSSI with the following goals:

### Goals

 - Opening the door and identify room for collaboration and shared training events between organisations involved in research software training
 - Make training and workshop offers, further services and material discoverable
 - Share problems, lessons learned, plans and solutions
 - Connect and link learning resources from different stakeholders
 - Increase visibility of localized/regional efforts
 - Present training material and activities to other organizations
 - Exchange ideas

### Programme

| Presenter | Title | Slides |
|-----------|-------|--------|
| Fredo Erxleben (Helmholtz Federated IT Services, HIFIS), Stephanie Schworm (Helmholtz Information and Data Science Academy HIDA), Donatella Cea (Helmholtz AI) | Education and Training in HIDA, HIFIS and Helmholtz AI: Status Quo and Future Perspectives | LINK TO SLIDES |
| Peter Pelz (NFDI Section Training & Education) | Germany’s Research Data Management Infrastructure (NFDI) and its national approach to connect training offers, teaching resources and material | LINK TO SLIDES |

> | Lieke de Boer (Netherlands eScience Center) | Research Software Training NL - building training coordination network in the Netherlands | LINK TO SLIDES |

Author: ```suggestion
| Lieke de Boer (Netherlands eScience Center) | Research Software Training NL - building training coordination network in the Netherlands | [SLIDES](https://doi.org/10.5446/60137) |
```

> | Radovan Bast (CodeRefinery) | CodeRefinery  | LINK TO SLIDES |

Author: ```suggestion
| Matias Jääskeläinen (CodeRefinery) | CodeRefinery  | LINK TO SLIDES |
```
I guess it is better to add Matias as he was the speaker?

Author: ```suggestion
| Matias Jääskeläinen (CodeRefinery) | CodeRefinery  | [SLIDES](https://doi.org/10.5446/60140) |
```

> | Michel Hernandez Villanueva (DESY and HSF) | Training Activities and Experiences in the HSF | LINK TO SLIDES |

Author: ```suggestion
| Michel Hernandez Villanueva (DESY and HSF) | Training Activities and Experiences in the HSF | [SLIDES](https://doi.org/10.5446/60138) |
```

| Danielle Sieh (The Carpentries) | The Carpentries - plans and implementation for signing-up for workshops with AMY, post teaching opportunities, match skills, interest, and availability | LINK TO SLIDES |

### Round table discussion

> SUMMARY OF DISCUSSION ?

Author: ```suggestion
### Summary of discussion
During the session it was clear that all participants share a common issue: training opportunities for RSE skills are in high demand and there is a big need for more trainings and more instructors. It was clear that there is a need for more coordination between organizations that provide trainings across Europe. Reusing training materials, automating workshop organisation, sharing experiences, are just some ideas on how this cooperation could benefit all parties involved.

You can also read the [HIFIS at the EuSSI training bazar](https://hifis.net/news/2022/11/30/HIFIS-at-EuSSI-training-bazar.html) blog post!
```
